packages/voting/lib/queries.js

```diff
@@ -183,7 +211,8 @@ const buildQueries = (tableName) => {
     numSubmitted,
     numEligible,
     isEligible,
-    turnout
+    turnout,
+    ensureReadyToSubmit
```

Comment:
I partly agree with your judgement. Yes it's a set of common-libs for votings, elections and qustionnaires. And all of the code applies equally to all of them, and the `tableMapping` defines the shared structure. It would be possible to better modularize them into a smaller files, totaly agree, but as it's not that long I prefer to let it be as it is.